topics:
    odom: '/agent/odom'
    laser_scan: '/scan'
    rgb_image: '/agent/camera/rgb/image_raw'
    agent_cmd: '/agent/cmd_vel'
    gazebo_states: '/gazebo/model_states'
    gazebo_set: '/gazebo/set_model_state'
    
params:
    laser_size: 360
    img_size: 80
    num_sikp_frame: 2
    num_stack_frame: 4
    reward_near_goal: 1000
    reward_near_obs: -10

dist:
    laser_sacn_clip: 5.0
    near_goal: 0.5
    near_obs: 0.7 # obs_dist_safe
    min_scan: 0.3 # min_sca

learning:
    learning_rate: 0.01
    gamma: 0.9
    replace_target_iter: 200
    memory_size: 5000 
    num_episode: 10000